pkg/inject/oauth-proxy.go

```diff
@@ -32,18 +32,22 @@ func OAuthProxy(jaeger *v1.Jaeger, dep *appsv1.Deployment) *appsv1.Deployment {
 }
 
 func getOAuthProxyContainer(jaeger *v1.Jaeger) corev1.Container {
+	// keep this sorted!
+	// see https://github.com/jaegertracing/jaeger-operator/pull/337
+	args := []string{
+		"--cookie-secret=SECRET",
+		"--https-address=:8443",
+		fmt.Sprintf("--openshift-service-account=%s", account.OAuthProxyAccountNameFor(jaeger)),
+		"--provider=openshift",
+		"--tls-cert=/etc/tls/private/tls.crt",
+		"--tls-key=/etc/tls/private/tls.key",
+		"--upstream=http://localhost:16686",
+	}
+
```

Comment:
Would it be better to add the sort, so not relying on users to read the comment?

Reply:
Although I've just seen the test, so that should pick it up.

Reply:
Thought about it, but as this is a static list, it might be better to enforce via test and spare a couple of CPU cycles :-)